docs/tutorials/multiplex_analysis.md

```diff
@@ -49,9 +49,9 @@ The *Fluorescence* type here tells QuPath that 'high pixel values mean more of s
 Choosing *Brightfield* conveys the opposite message, which would cause problems because cell detection would then switch to looking for dark nuclei on a light background.
 :::
 
-:::{sidebar} Accurate cell detection
-Good cell segmentation is really *essential* for accurate multiplexed analysis.
-New and improved methods of segmenting cells in QuPath are being actively explored...
+:::{sidebar} Precise cell detection
```

Comment:
I don't see a reason to change this, I think 'Accurate' more accurately describes what is intended here - https://en.wikipedia.org/wiki/Accuracy_and_precision